lib/redbird/key.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Redbird.Key do
+  alias Redbird.Crypto
+
+  def generate(namespace \\ namespace()) do
+    namespace <> (96 |> :crypto.strong_rand_bytes() |> Base.encode64())
```

Comment:
Do you know why we use `96` here? I know that was something this library already did. But if you figured out why, it might be nice to extract it into a module attribute an name it something helpful. As it is right now, it's a bit of a mystery guest.

Reply:
I don't know why `96`  was chosen specifically. I will move it to a named module attribute though to add some clarity.